packages/dev/docs/pages/releases/2023-04-04.mdx

```diff
@@ -0,0 +1,96 @@
+{/* Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {BlogPostLayout, Hero} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+description: In this release, we have many updates for our pre-release components including TagGroup and Table column resizing, and Avatar has moved to GA. We are also releasing the first alpha of React Aria Components, brand new component library built on top of our React Aria hooks! It provides a simpler way to build accessible components with custom styles, while offering the flexibility to drop down to hooks for even more customizability where needed.
+date: 2023-04-04
+---
+
+# April 4, 2023 Release
+
+In this release, we have many updates for our pre-release components including TagGroup and Table column resizing, both of which have reached RC status. Avatar has also moved to GA. We also included some API improvements to our existing React Aria hooks, fixed some SSR issues, improved overlay positioning, and fixed many other bugs.
+
+We are also releasing the first alpha of [React Aria Components](../../react-aria/react-aria-components.html), a brand new component library built on top of our React Aria hooks! It provides a simpler way to build accessible components with custom styles, while offering the flexibility to drop down to hooks for even more customizability where needed. It includes a default DOM structure and styling API, and abstracts away all of the glue code necessary to connect the hooks together, allowing you to focus more on your unique design and styling. Check out the [documentation](../../react-aria/react-aria-components.html) to learn more, and read our [RFC](https://github.com/adobe/react-spectrum/blob/main/rfcs/2023-react-aria-components.md) for even more details. While it is in alpha, we'd love for you to try it out and [give us feedback](https://github.com/adobe/react-spectrum/issues) on the APIs so we can shape it into the best library it can be!
```

Comment:
We don't usually call out alphas this prominently, but RAC is a pretty big new feature and I think it would be good to get feedback on the APIs. Thoughts?

Reply:
Makes sense to me. Would also be good to call out the alpha release in the original [discussion](https://github.com/adobe/react-spectrum/discussions/2368) + Twitter (though I'm sure you were already planning to do those haha)